src/Datatable/Datatable.php

```diff
@@ -76,6 +76,7 @@ class Datatable
             'headersAttrsTr' => [],
             'headersAttrsTh' => [],
         ],
+		'multiSelectType' => 'jquery-ui',
```

Comment:
set select2 as default

Reply:
add constants to this same class to set these values
ex: 
```
const MULTI_SELECT_TYPE_SELECT2 = 'select2';
const MULTI_SELECT_TYPE_JQUERY_UI = 'jquery-ui';
```

Reply:
Good idea, added constants and set select 2 as default.